app/SearchApi/SearchRequest.Adaptor/Consumer/SearchRequestOrderedConsumer.cs

```diff
@@ -30,9 +31,17 @@ public async Task Consume(ConsumeContext<SearchRequestOrdered> context)
             using (LogContext.PushProperty("RequestRef", $"{context.Message?.Person?.Agency?.RequestId}"))
             using (LogContext.PushProperty("AgencyCode", $"{context.Message?.Person?.Agency?.Code}"))
             {
-                _logger.LogInformation("get the searchRequestOrdered message.");
-                var cts = new CancellationTokenSource();
-                await _searchRequestNotifier.NotifySearchRequestEventAsync(context.Message.RequestId, context.Message, cts.Token, context.GetRetryAttempt(), _retryConfig.Value.RetryTimes);
+                try
+                {
+                    _logger.LogInformation("get the searchRequestOrdered message.");
+                    var cts = new CancellationTokenSource();
```

Comment:
Since you are not cancelling the token source, you can use CancellationToken.None to avoid allocoations

Reply:
ok